### ISSUE TITLE
Rename npm package to contributte

### ISSUE THREAD

**Issue by adaamz** [contributte/datagrid#898]
Hi, I just find out that there is still old named npm package `ublaboo-datagrid`. What about `contributte-datagrid`?

**Comment by f3l1x** [comment]
Good point. We have already team at npm. Maybe `@contributte/datagrid`.
